lib/vm_service_wrapper.dart

```diff
@@ -0,0 +1,268 @@
+  Future<Breakpoint> addBreakpointAtEntry(
+          String isolateId, String functionId) =>
+      _trackFuture(_vmService.addBreakpointAtEntry(isolateId, functionId));
+
+  @override
+  Future<Breakpoint> addBreakpointWithScriptUri(
+          String isolateId, String scriptUri, int line, {int column}) =>
+      _trackFuture(_vmService.addBreakpointWithScriptUri(
+          isolateId, scriptUri, line,
+          column: column));
+
+  @override
+  Future<Response> callMethod(String method, {String isolateId, Map args}) =>
+      _trackFuture(
+          _vmService.callMethod(method, isolateId: isolateId, args: args));
+
+  @override
+  Future<Response> callServiceExtension(String method,
+          {String isolateId, Map args}) =>
+      _trackFuture(_vmService.callServiceExtension(method,
+          isolateId: isolateId, args: args));
+
+  @override
+  Future<Success> clearCpuProfile(String isolateId) =>
+      _trackFuture(_vmService.clearCpuProfile(isolateId));
+
+  @override
+  Future<Success> clearVMTimeline() =>
+      _trackFuture(_vmService.clearVMTimeline());
+
+  @override
+  Future<Success> collectAllGarbage(String isolateId) =>
+      _trackFuture(_vmService.collectAllGarbage(isolateId));
+
+  @override
+  void dispose() => _vmService.dispose();
+
+  @override
+  Future evaluate(String isolateId, String targetId, String expression,
+          {Map<String, String> scope}) =>
+      _trackFuture(
+          _vmService.evaluate(isolateId, targetId, expression, scope: scope));
+
+  @override
+  Future evaluateInFrame(String isolateId, int frameIndex, String expression,
+          {Map<String, String> scope}) =>
+      _trackFuture(_vmService.evaluateInFrame(isolateId, frameIndex, expression,
+          scope: scope));
+
+  @override
+  Future<AllocationProfile> getAllocationProfile(String isolateId,
+          {String gc, bool reset}) =>
+      _trackFuture(_vmService.getAllocationProfile(isolateId));
+
+  @override
+  Future<CpuProfile> getCpuProfile(String isolateId, String tags) =>
+      _trackFuture(_vmService.getCpuProfile(isolateId, tags));
+
+  @override
+  Future<FlagList> getFlagList() => _trackFuture(_vmService.getFlagList());
+
+  @override
+  Future<ObjRef> getInstances(String isolateId, String classId, int limit) =>
+      _trackFuture(_vmService.getInstances(isolateId, classId, limit));
+
+  @override
+  Future getIsolate(String isolateId) =>
+      _trackFuture(_vmService.getIsolate(isolateId));
+
+  @override
+  Future getObject(String isolateId, String objectId,
+          {int offset, int count}) =>
+      _trackFuture(_vmService.getObject(isolateId, objectId));
+
+  @override
+  Future<ScriptList> getScripts(String isolateId) =>
+      _trackFuture(_vmService.getScripts(isolateId));
+
+  @override
+  Future<SourceReport> getSourceReport(
+          String isolateId, List<SourceReportKind> reports,
+          {String scriptId,
+          int tokenPos,
+          int endTokenPos,
+          bool forceCompile}) =>
```

Comment:
bonus points: add a trailing comma to all the method signatures that wrap. It makes the code a bit easier to read.

Reply:
done.